RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -362,6 +362,14 @@ boolean LX_unlockHauntedLibrary()
 	{
 		auto_log_info("I don't need to equip a cue to beat this ghostie.", "blue");
 	}
+	else if(in_glover()) // can't use pool cues or benefit from most of the pool effects
+	{
+		if(possessEquipment($item[government-issued eyeshade]))
+		{
+			autoEquip($item[government-issued eyeshade]);
+			expectPool += 5;
+		}
+	}
 	else
```

Comment:
I didn't make this conditional upon just an auto_is_valid check, since in most other paths we shouldn't have trouble meeting the pool skill requirement in the first place. Maybe that would be better though.